src/Lithnet.GoogleApps/GroupRequestFactory.cs

```diff
@@ -113,7 +116,7 @@ private void PopulateGroups(string customerID, string userKey, string groupField
                     {
                         request.PageToken = token;
 
-                        Groups pageResults = request.ExecuteWithRetryOnBackoff();
+                        Groups pageResults = request.ExecuteWithRetry(RetryEvents.Backoff | RetryEvents.Timeout, ApiExtensions.RetryCount);
```

Comment:
The ApiExtensions.RetryCount parameter is not needed here, as the appropriate overload of ExecuteWithRetry will default to this parameter

Reply:
My bad, I mistook the consumeTokens for the retrycount. Changing.